swift/test/Ice/udp/AllTests.swift

```diff
@@ -48,11 +36,6 @@ public func allTests(_ helper: TestHelper) async throws {
     let communicator = helper.communicator()
     communicator.getProperties().setProperty(key: "ReplyAdapter.Endpoints", value: "udp")
     let adapter = try communicator.createObjectAdapter("ReplyAdapter")
```

Comment:
How do we know this test will not use collocation? The reply object and proxy share the same communicator.

Reply:
In these UDP tests, we pass the reply proxy as an argument to `ping`, and it's the ping implementation in the server  (TestI.swift) that calls this proxy. There is no collocation concern.